plot/plot.py

```diff
@@ -147,7 +153,7 @@ def save_split_vln_plot(df, out_path, ymax):
 
     ax.legend()  # Seems to have the effect of removing the title of the legend
     handles, labels = ax.get_legend_handles_labels()
-    ax.legend(handles, ["Before 2021", "2021"], loc="upper left")
+    ax.legend(handles, ["Before " + YEAR_TO_SPLIT_STR, YEAR_TO_SPLIT_STR + "+"], loc="upper left")
```

Comment:
```suggestion
    ax.legend(handles, [f"Before {YEAR_TO_SPLIT}", f"{YEAR_TO_SPLIT_STR}+"], loc="upper left")
```

Using f-strings would remove the need for a separate `YEAR_TO_SPLIT_STR` and IMO are more readable